cypress/support/cypress-commands/assertion.js

```diff
@@ -512,7 +512,10 @@ Cypress.Commands.add(
             JSON.stringify(extractEventObject),
             CONSTANTS.PASS,
```

Comment:
![image](https://github.com/rdkcentral/firebolt-certification-suite/assets/148241508/26aa7285-5ea7-4718-a6e9-a38cc839e4cf)
Do we need to make this change ? will it work without this ?

Reply:
json parse can cause an issue, but json stringify can be done,